lib/components/output-area.js

```diff
@@ -18,20 +20,23 @@ const EmptyMessage = () => {
   );
 };
 
-const OutputArea = observer(({ store: { kernel } }: { store: store }) => {
-  if (!kernel) {
-    if (atom.config.get("Hydrogen.outputAreaDock")) {
-      return <EmptyMessage />;
-    } else {
-      atom.workspace.hide(OUTPUT_AREA_URI);
-      return null;
-    }
-  }
+@observer
+class OutputArea extends React.Component<{ store: store }> {
+  showHistory: IObservableValue<boolean> = observable(true);
+  @action
+  setHistory = () => {
+    this.showHistory.set(true);
+  };
+
+  @action
+  setScrollList = () => {
+    this.showHistory.set(false);
+  };
 
-  const handleClick = () => {
+  handleClick = () => {
+    const kernel = this.props.store.kernel;
     if (!kernel || !kernel.outputStore) return;
-    //convert output to a plain js object
-    const output = toJS(kernel.outputStore.outputs[kernel.outputStore.index]);
```

Comment:
I've been testing this just now and have not had any issues without the `toJS` call :smile: